mkosi

```diff
@@ -1028,7 +1028,7 @@ def install_opensuse(args, workspace, run_build_script):
     #
     # Install the "minimal" package set.
     #
-    subprocess.run(cmdline + ["-t", "pattern", "minimal_base"], check=True)
+    subprocess.run(cmdline + ["patterns-base-minimal_base"], check=True)
```

Comment:
no sorry, I reported the issue internally and apparently it's not fixed yet. OTOH, I would suggest you to open a bug if you want to track it down.